lighthouse-core/test/gather/gatherers/link-elements-test.js

```diff
@@ -31,7 +31,7 @@ describe('Link Elements gatherer', () => {
   function getPassData({linkElementsInDOM = [], headers = []}) {
     const url = 'https://example.com';
     const loadData = {networkRecords: [{url, responseHeaders: headers, resourceType: 'Document'}]};
-    const driver = {evaluate: () => Promise.resolve(linkElementsInDOM)};
+    const driver = {executionContext: {evaluate: () => Promise.resolve(linkElementsInDOM)}};
```

Comment:
I'm kinda on the fence but I'm thinking expanding this over a few lines would help for readability.

Reply:
sure